sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -157,19 +157,6 @@ class SQLContext private[sql](val sparkSession: SparkSession)
     sparkSession.conf.getAll
   }
 
-  /**
-   * :: Experimental ::
-   * A collection of methods that are considered experimental, but can be used to hook into
-   * the query planner for advanced functionality.
-   *
-   * @group basic
-   * @since 1.3.0
-   */
-  @Experimental
-  @transient
-  @Unstable
-  def experimental: ExperimentalMethods = sparkSession.experimental
```

Comment:
I would avoid the breaking change.

Reply:
@HyukjinKwon Thanks for your review. I will close this PR.